pages/about/translating/wcag.md

```diff
@@ -131,7 +132,27 @@ Download [`fixup.js` {% include_cached external.html %}](https://www.w3.org/scr
 - "Pop Out Sidebar"
 - "Jump to Table of Contents”
 
-## Step 5: Deliver the final files to W3C {#deliver-files}
+## Step 5: Check your translation thoroughly {#check}
+
+Please properly check your files before submission to W3C.
+
+In particular, make sure that:
+- All links are valid. You can use [W3C Link Checker](https://validator.w3.org/checklink)
+- HTML is valid. You can use [W3C Markup Validation Service](https://validator.w3.org/)
+- CSS is valid. You can use [W3C CSS Validation Service](https://jigsaw.w3.org/css-validator/)
+- Pages are internationalisation-friendly. You can use [W3C Internationalization Checker](https://validator.w3.org/i18n-checker/)
+- All relevant content has been properly translated
+- All `id`s are in English (= they have not been translated).
```

Comment:
how about:
> (= they have not been translated)

->
; that is, they have not been translated.

Reply:
hey @remibetin For things that they should not do like translate IDs and add custom CSS, are those clearly explained early in the process? (We don't want them to have to undo that work!)

Reply:
> ; that is, they have not been translated.

Fixed.

> For things that they should not do like translate IDs and add custom CSS, are those clearly explained early in the process? (We don't want them to have to undo that work!)

I have added further indication.